pkgs/applications/networking/instant-messengers/chatterino2/default.nix

```diff
@@ -1,25 +1,33 @@
-{ mkDerivation, lib, pkgconfig, fetchFromGitHub, qtbase, qtsvg, qtmultimedia, qmake, boost, openssl }:
+{ mkDerivation, stdenv, lib, pkgconfig, fetchFromGitHub, qtbase, qtsvg, qtmultimedia, qmake, boost, openssl, wrapQtAppsHook }:
 
 mkDerivation rec {
   pname = "chatterino2";
-  version = "unstable-2019-05-11";
+  version = "2.1.7";
   src = fetchFromGitHub {
     owner = "fourtf";
     repo = pname;
-    rev = "8c46cbf571dc8fd77287bf3186445ff52b1d1aaf";
-    sha256 = "0i2385hamhd9i7jdy906cfrd81cybw524j92l87c8pzrkxphignk";
+    rev = "v${version}";
+    sha256 = "0bbdzainfa7hlz5p0jfq4y04i3wix7z3i6w193906bi4gr9wilpg";
     fetchSubmodules = true;
   };
-  nativeBuildInputs = [ qmake pkgconfig ];
+  nativeBuildInputs = [ qmake pkgconfig wrapQtAppsHook ];
```

Comment:
This wasn't resolved, nor was the first one. It's unnecessary clutter to specify `wrapQtAppsHook` because it is automatically added when using the qt-specific deriver.

https://nixos.org/nixpkgs/manual/#sec-language-qt

 `wrapQtAppsHook` should be specified iff you cannot use the qt-specific deriver: "**If you cannot use mkDerivation** or mkDerivationWith above, include wrapQtAppsHook in nativeBuildInputs" (emphasis mine).